launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -323,6 +341,40 @@ public SparkLauncher setVerbose(boolean verbose) {
    * @return A process handle for the Spark app.
    */
   public Process launch() throws IOException {
+    return createBuilder().start();
+  }
+
+  /**
+   * Starts a Spark application.
+   * <p/>
+   * This method returns a handle that provides information about the running application and can
+   * be used to do basic interaction with it.
+   *
+   * @since 1.5.0
+   * @param listeners Listeners to add to the handle before the app is launched.
+   * @return A handle for the launched application.
```

Comment:
can you add a comment on `launch()` that `startApplication()` is generally preferred?